source/documentation/virtual_machine_management_guide/topics/Creating_a_snapshot_of_a_virtual_machine.adoc

```diff
@@ -2,11 +2,17 @@
 [id="Creating_a_snapshot_of_a_virtual_machine"]
 = Creating a Snapshot of a Virtual Machine
 
-A snapshot is a view of a virtual machine's operating system and applications on any or all available disks at a given point in time. Take a snapshot of a virtual machine before you make a change to it that may have unintended consequences. You can use a snapshot to return a virtual machine to a previous state.
+A snapshot is a view of a virtual machine's operating system and applications on any or all available disks at a given point in time. Take a snapshot of a virtual machine before you make a change to it that may have unintended consequences. You can use a snapshot to restore a virtual machine to a previous state.
 
 
 *Creating a Snapshot of a Virtual Machine*
 
+.In the VM Portal:
+. Open a virtual machine.
+. In the *Snapshots* panel, click btn:[+Create Snapshot].
+. A snapshot is added to the panel, including all attached disks.
```

Comment:
Does the system response get a number? I usually write:

[Step number] . Do something.
 [+]                       The system does what it is supposed to.